docs/README.md

```diff
@@ -1,18 +1,19 @@
+
```

Comment:
Why this change?

Reply:
No problem, I was just wondering if it was done in purpose :slightly_smiling_face: